tests/php/TestHealthCheckElasticsearch.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Test health check elasticsearch functionality.
+ *
+ * @since 4.4.1
+ * @package elasticpress
+ */
+
+namespace ElasticPressTest;
+
+use ElasticPress\Elasticsearch;
+use WP_Site_Health;
+use WP_Ajax_UnitTestCase;
+use WPAjaxDieContinueException;
+
+/**
+ *  Health check elasticsearch test class
+ */
+class TestHealthCheckElasticsearch extends WP_Ajax_UnitTestCase {
+
+	/**
+	 * Test if the test is registered
+	 */
+	public function testIsRegistered() {
+		$tests = WP_Site_Health::get_tests();
+
+		$this->assertArrayHasKey( 'elasticpress-health-check-elasticsearch', $tests['async'] );
+	}
+
+	/**
+	 * Test ajax output.
+	 */
+	public function testAjaxOutput() {
+		$admin_id = $this->factory->user->create( array( 'role' => 'administrator' ) );
+		wp_set_current_user( $admin_id );
+
+		// Make the request.
+		try {
+			$this->_handleAjax( 'health-check-elasticpress-health-check-elasticsearch' );
+		} catch ( WPAjaxDieContinueException $e ) {
+			unset( $e );
+		}
+
+		$response = json_decode( $this->_last_response, true );
+
+		$this->assertEquals( true, $response['success'] );
```

Comment:
Can we change this to
```suggestion
		$this->assertTrue( $response['success'] );
```

Reply:
Done